src/main/java/com/denizenscript/depenizen/bukkit/commands/quests/QuestsCommand.java

```diff
@@ -0,0 +1,221 @@
+package com.denizenscript.depenizen.bukkit.commands.quests;
+
+import com.denizenscript.denizencore.objects.Argument;
+import com.denizenscript.denizencore.objects.ArgumentHelper;
+
+import me.blackvein.quests.Quest;
+import me.blackvein.quests.Quester;
+import me.blackvein.quests.Quests;
+import me.blackvein.quests.exceptions.InvalidStageException;
+import me.blackvein.quests.util.Lang;
+
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+
+import org.bukkit.entity.Player;
+
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.debugging.Debug;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsException;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.depenizen.bukkit.bridges.QuestsBridge;
+
+/**
+ * Command integration for Quests
+ * @author PikaMug
+ * @since 2.0.0-b595
+ */
+public class QuestsCommand extends AbstractCommand {
+
+    // <--[command]
+    // @Name Quests
+    // @Syntax quests [add/remove/set] (questId:<quest-id>) (stage-no:<#>) (points:<#>) (state:true/false)

```

Comment:
You can do `questid`, or `quest_id`... not "questId" (no capitals).
However, realistically, just go with `quest`
Similar, `stage-no` is a bit ... off. `stage` would be better.

Reply:
Yes, I found out about "questId" the hard way - forgot to remove that remnant. I agree that underscores look better, but merely using `quest` would imply that the user-friendly quest name is acceptable. Since said names may contain spaces, we must use the ID to conform to syntax.

Simialrly for stages, it must be the numerical stage and not some other identifier.

Reply:
`-no` and `-id` are nonsense. `quest:<id>` and `stage:<#>` both clearly express the requirements you listed, and further explanation or clarity can be included in the description block.

Reply:
I feel the problem stems from how Quests issues IDs (something the original author designed). IDs are not strictly numerical and are issued as a string to the tune of "custom1", "custom2" ... "custom100" etc. The server operator is never exposed to these IDs until they look into the quests.yml file, as they were never meant to be edited. Thus, I find it reasonable to think that anyone whom does not read the command description might assume that `quest:custom1` refers to a name. They would then create a new command such as `quest:My First Quest` which would fail, much to their confusion.

It's my intention to expose the IDs in a future version of Quests (without the ability to edit), but even then it will take some before users are able to competently identify the difference between them and names. This is why I took the extra precaution of appending `-id` (now `_id`) even though it isn't as pretty. I hope that it will encourage users to pause and check before potentially making the wrong assumption.

Same goes for `stage_no` as far as confusing it with an objective name, although this is far less likely as stage numbers are already exposed.

Reply:
Changed `stage_no` to `stage`